tests/js/browser-click-tests.js

```diff
@@ -0,0 +1,121 @@
+/*
+
+  Test clicking on various page elements.
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("../../index");
+gpii.tests.browser.loadTestingSupport();
+
+var url      = require("url");
+var startUrl = url.resolve(url.resolve("file://", __dirname), "./static/html/click.html");
```

Comment:
Note that the syntax used with the KETTLE-32 branch has changed to %module rather than ${module)